### ISSUE TITLE
[Bug] Crash Aggregator container

### ISSUE THREAD

**Issue by haooliveira84** [kubecost/features-bugs#116]
### Kubecost Version

2.3.3

### Kubernetes Version

1.29

### Kubernetes Platform

EKS

### Description

Aggregator pod restart every moment.

I've seem this error in log

```sh
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x2c0f5c0]
goroutine 1188 [running]:
github.com/kubecost/kubecost-cost-model/pkg/duckdb/internal/bun.(*Writer).InsertCloudCosts(0x4003d51cc0, {0x692ce68, 0x40017e6090}, 0x4001597110, 0x4002640000)
	/app/kubecost-cost-model/pkg/duckdb/internal/bun/writer.go:698 +0x160
github.com/kubecost/kubecost-cost-model/pkg/duckdb/internal/cloudcost.(*Ingestor).run.func1(0x4001597110)
	/app/kubecost-cost-model/pkg/duckdb/internal/cloudcost/ingestor.go:247 +0xd88
github.com/opencost/opencost/core/pkg/util/worker.(*queuedWorkerPool[...]).worker(0x0)
	/app/opencost/core/pkg/util/worker/worker.go:117 +0x50
created by github.com/opencost/opencost/core/pkg/util/worker.NewWorkerPool[...] in goroutine 990
	/app/opencost/core/pkg/util/worker/worker.go:72 +0x13c
```

### Steps to reproduce

1. Uses kubecost

### Expected behavior

Agregator running by complete

### Troubleshooting

- [X] I have read and followed the [issue guidelines](https://github.com/kubecost/features-bugs/blob/main/ISSUE_GUIDELINES.md) and this is a bug impacting only the Kubecost application.
- [X] I have searched other issues in this repository and mine is not recorded.

**Comment by cliffcolvin** [comment]
@haooliveira84 I'm looking into this with my team. Will update soon.

**Comment by cliffcolvin** [comment]
@haooliveira84 this should be available in our 2.3.5 patch going out this week. I'll be cutting a new release candidate for this as well today sometime, if you're interested in a release candidate to fix it.
